source/includes/read/change-streams.rb

```diff
@@ -0,0 +1,56 @@
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string>'
+
+Mongo::Client.new(uri) do |client|
+  # Accesses the database and collection
+  # start-db-coll
+  database = client.use('sample_restaurants')
+  collection = database[:restaurants]
+  # end-db-coll
+  
+  # Monitors and prints changes to the "restaurants" collection
+  # start-open-change-stream
+  stream = collection.watch
+  enum = stream.to_enum
+  while (doc = enum.next)
+    puts doc
+    break if doc['operationType'] == 'invalidate'
+  end
```

Comment:
Using `stream#each` will be simpler, and more idiomatic:

```suggestion
  stream.each do |doc|
    puts doc
    break if doc['operationType'] == 'invalidate'
  end
```